ThinkerCAD.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=ThinkerCAD Dekstop
+Exec=AppRun --no-sandbox %U
```

Comment:
this is INCORRECT. It should match what's in `command` + the eventual argument placeholders.